### PR TITLE
Flatten migrations

### PR DESCRIPTION
**Pull request by howard-e**
The migrations used in this project have grown as much as "deemed necessary" over the project's lifespan. This PR aims to remove those migrations and preserve the structure of the tables. Due to the suggested point for this PR's merge, there shouldn't be a need to preserve any functional changes of those previous migrations as all environments should be up to date migration-wise and the migrations were mostly focused on keeping up to date with changes to the application's code (from what I've gone through at least).

This should present a fresh start on the migrations by limiting the time taken to run them and also attempt to maintain the practice of, as best as possible to avoid using application code in any migration going forward.

### CODE REVIEW

File: server/migrations/utils/dumpTable.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const dumpTable = async tableName => {
+  try {
+    const dumpFilePath = path.resolve(
+      `${__dirname}/../dumps/dumpTable_${tableName}_${new Date().getTime()}.sql`
+    );
+
+    await exec(
+      `pg_dump -t '"${tableName}"' ${process.env.PGDATABASE} --inserts > ${dumpFilePath}`
+    );
+  } catch (err) {
```
Comment (stalgiag) [inline server/migrations/utils/dumpTable.js:20]: Do we want to catch this? Might be safer to not run the migration if making the dump fails.

**Review by stalgiag — COMMENTED** [review]
This is a terrific change! Left some minor comments. The main concern I would have is with continuing the migration after a failed backup dump.

File: server/migrations/utils/dumpTable.js (hunk 1 above)
Comment (howard-e) [reply server/migrations/utils/dumpTable.js:19]: Good point! Addressed in https://github.com/w3c/aria-at-app/pull/1187/commits/72861c2854fa6867767309b03ae27b8255504628

File: server/migrations/utils/dumpTable.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * Used to dump a table in case a significantly destructive behavior is to be
+ * done. This should make running a down migration possible if required.
+ *
+ * @param tableName
+ * @returns {Promise<void>}
+ */
+const dumpTable = async tableName => {
+  try {
+    const dumpFilePath = path.resolve(
+      `${__dirname}/../dumps/dumpTable_${tableName}_${new Date().getTime()}.sql`
```
[the remark this answers is not in the document]
Comment (howard-e) [reply server/migrations/utils/dumpTable.js:14]: I think I'd rather this be left to each contributor to decide how to handle for themselves. I suppose utils for `clean:migrationDump:tables` or `clean:migrationDump:databases` could be added but I don't think it's super necessary (to me)

File: server/migrations/utils/index.js
[the remark this answers is not in the document]
Comment (howard-e) [reply server/migrations/utils/index.js:1]: > I am going to assume no functional changes

That's right!

File: server/migrations/utils/dumpTable.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (stalgiag) [reply server/migrations/utils/dumpTable.js:14]: Makes sense! No need for a util for now.

**Review by stalgiag — APPROVED** [review]
Great! Thanks for addressing the comments.

**Pull request merged**